### PR TITLE
Fix the `vagrant up` situation

### PR DESCRIPTION
**Pull request by ghost**
Cloning the repository and issuing a `vagrant up` resulted in scary red fail messages. These commits tidy that situation up a little bit.

Vagrantfile has also been cleaned up to look more like the one `meez` creates, and extended to silence the annoying ssl_verify_mode message.

### CODE REVIEW

**Comment by martinb3** [comment]
For what it's worth, we're not really supporting Vagrant directly in the stacks these days. Test-kitchen creates a Vagrant file on the fly, and we're making most adjustments in the TK config. And while we're not breaking things for chef-solo, it's going to be really painful to test with, as many of the recipes expect LOTS of explicit values when running under chef-solo.

I'm actually surprised there was a Vagrantfile in here at all.

**Comment by ghost** [comment]
It's cleaner now :)

File: recipes/logging.rb [hunk 1]
```diff
@@ -17,8 +17,9 @@
   Chef::Log.warn('Cannot invoke search for ELK cluster while running under chef-solo')
 else
   include_recipe 'elasticsearch::search_discovery'
+  elk_nodes = node['elasticsearch']['discovery']['zen']['ping']['unicast']['hosts']
 end
-elk_nodes = node['elasticsearch']['discovery']['zen']['ping']['unicast']['hosts']
+
 found_elkstack = !elk_nodes.nil? && !elk_nodes.split(',').empty? # don't do anything unless we find nodes
```
Comment (martinb3) [inline recipes/logging.rb:23]: Did you want to move the elk_nodes variable as well? I think I changed that so people could override it if they didn't have chef-zero vs. just breaking.

**Comment by ghost** [comment]
I dunno, it broke with Vagrant+Chef Solo, looked related to elasticsearch::search_discovery, so I put it in the conditional. People can still set it instead of override it, right?

**Comment by martinb3** [comment]
No, moving it inside the else means it won't run on chef-solo, preventing someone from providing an alternate value for `elk_nodes` by populating `node['elasticsearch']['discovery']['zen']['ping']['unicast']['hosts']`.

``` ruby
if Chef::Config[:solo] # only runs on solo
  Chef::Log.warn('Cannot invoke search for ELK cluster while running under chef-solo')
else # only runs on chef that isn't solo
  include_recipe 'elasticsearch::search_discovery'
  + elk_nodes = node['elasticsearch']['discovery']['zen']['ping']['unicast']['hosts']
end
```

**Comment by ghost** [comment]
Oh, meh. I'll just wipe out the Vagrant stuff in another PR.

**Comment by nicolascayo** [comment]
what?

**Pull request closed without merging**